tests/integration/SdkTest_test.cpp

```diff
@@ -4853,27 +4847,34 @@ TEST_F(SdkTest, SdkBackupNames)
     ASSERT_EQ(MegaError::API_OK, err) << "setBackup failed (error: " << err << ")";
     ASSERT_TRUE(waitForResponse(&mApi[0].userUpdated));
 
-    MegaUser* u = megaApi[0]->getMyUser();
-    bool null_pointer = (u == nullptr);
-    ASSERT_FALSE(null_pointer) << "Cannot find the MegaUser for email: " << mApi[0].email;
-
     // retrieve the given backup name
     err = synchronousGetBackupName(0, mBackupId);
     ASSERT_EQ(MegaError::API_OK, err) << "getBackupName failed (error: " << err << ")"; 
-    ASSERT_EQ(mBackupName, backupName.get()) << "getBackupName returned incorrect value";
+    ASSERT_EQ(attributeValue, backupName.get()) << "getBackupName returned incorrect value";
 
     // give a new name to the backup
-    //err = synchronousSetBackupName(0, mBackupId, "BackupNamesTest");
-    //ASSERT_EQ(MegaError::API_OK, err) << "setBackupName failed (error: " << err << ")";
+    mApi[0].userUpdated = false;
+    backup = "/SdkBackupNames_setBackupName_Test";
+    err = synchronousSetBackupName(0, mBackupId, backup);
+    ASSERT_EQ(MegaError::API_OK, err) << "setBackupName failed (error: " << err << ")";
+    ASSERT_TRUE(waitForResponse(&mApi[0].userUpdated));
```

Comment:
You should not need to wait for response in this case. Once the `setBackupName()` finishes, the attribute is available (in contrast to the silent automatic setup on the background for the `setBackup()` call.